portal/README.md

```diff
@@ -1,3 +1,15 @@
-Portal
-======
-Block explorer web portal that allows you to explore and search CENNZnet blockchain for transactions, addresses, assets and other activities taking place on CENNZnet.
\ No newline at end of file
+# Portal
+
+Block explorer web portal that allows you to explore and search CENNZnet blockchain for transactions, addresses, assets and other activities taking place on CENNZnet.
+
+## development
+
+Contributions are welcome!
+
+To get started -
+
+1. Clone the repo locally, via `git clone git@github.com:cennznet/explorer.git <optional local path>`
+2. Change directory to the portal project, via `cd portal`
+3. Install the dependencies by running `yarn`
+4. Run in the dev mode, via `yarn run dev`
+5. Access the app via [http://localhost:3000](http://localhost:3000)
```

Comment:
1.  git clone git@github.com:cennznet/explorer.git <optional local path> : clone the repo locally
2.  cd portal : change to the portal project directory
3.  yarn : Install the dependencies
4.  yarn run dev: run in dev mode
5.  [http://localhost:3000](http://localhost:3000) : open the app